### DISCUSSION TITLE
Intruducing SLSA: Supply chain Levels for Software Artifacts

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#81] [日常讨论]
**Securing the Software Supply Chain with SLSA**: https://thenewstack.io/securing-the-software-supply-chain-with-slsa/

[Introducing SLSA, an End-to-End Framework for Supply Chain Integrity](https://security.googleblog.com/2021/06/introducing-slsa-end-to-end-framework.html)

官网：https://slsa.dev/

![image](https://user-images.githubusercontent.com/15010826/167523076-fae32837-c37d-43a1-a529-3247d91b64a9.png)

**Comment by wangleo61** [comment]
Looks very nice

Leo
发件人: Will ***@***.***>
发送时间: Tuesday, May 10, 2022 9:14 AM
收件人: ***@***.***>
抄送: ***@***.***>
主题: [kaiyuanshe/ONES] Intruducing SLSA: Supply chain Levels for Software Artifacts (Discussion #81)


Securing the Software Supply Chain with SLSA: https://thenewstack.io/securing-the-software-supply-chain-with-slsa/

Introducing SLSA, an End-to-End Framework for Supply Chain Integrity<https://security.googleblog.com/2021/06/introducing-slsa-end-to-end-framework.html>

官网：https://slsa.dev/

[image]<https://user-images.githubusercontent.com/15010826/167523076-fae32837-c37d-43a1-a529-3247d91b64a9.png>

―

**Reply by will-ww** [reply]
> Looks very nice Leo 发件人: Will ***@***.***> 

你这速度，飞快~

SLSA 是个好东西，把软件供应链的术语、框架、体系，打了个样，非常值得我们参考~

**Comment by wangleo61** [comment]
王老师太厉害了，都能找到这么好的东东，空了研究一下，这个协助效率好高啊，好爽呀

Leo

发件人: Will ***@***.***>
发送时间: Tuesday, May 10, 2022 9:22 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Intruducing SLSA: Supply chain Levels for Software Artifacts (Discussion #81)


Looks very nice Leo 发件人: Will @.***>

你这速度，飞快~

SLSA 是个好东西，把软件供应链的术语、框架、体系，打了个样，非常值得我们参考~

―

**Reply by will-ww** [reply]
看有谁愿意来分享的，没有后面就安排我们同学来分享下了~

**Comment by wangleo61** [comment]
我愿意来分享哈，先报个名先

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Tuesday, May 10, 2022 9:30 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Intruducing SLSA: Supply chain Levels for Software Artifacts (Discussion #81)


看有谁愿意来分享的，没有后面就安排我们同学来分享下了~

**Reply by will-ww** [reply]
好咧，安排上了~

**Comment by wangleo61** [comment]
哈哈哈，谢谢王老师

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Tuesday, May 10, 2022 9:33 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Intruducing SLSA: Supply chain Levels for Software Artifacts (Discussion #81)


好咧，安排上了~

―
